src/python/grpcio/grpc/__init__.py

```diff
@@ -926,6 +930,15 @@ def stream_stream(self,
         """
         raise NotImplementedError()
 
+    @abc.abstractmethod
+    def close(self):
```

Comment:
Please document it as idempotent: "closing a closed channel is a no-op" or something along that line.  

This is important because it is not just a feature to advertise, but to restrict all the implementers to ensure this method is indeed idempotent.

Reply:
I agree; done.